libafl/src/mutators/token_mutations.rs

```diff
@@ -1090,37 +1090,58 @@ impl AFLppRedQueen {
     }
 }
 
-impl<I, S> MultipleMutator<I, S> for AFLppRedQueen
+impl<I, S> Mutator<I, S> for AFLppRedQueen
+where
+    S: UsesInput + HasMetadata + HasRand + HasMaxSize + HasCorpus,
+    I: HasBytesVec + From<Vec<u8>>,
+{
+    fn mutate(
+        &mut self,
+        state: &mut S,
+        input: &mut I,
+        stage_idx: i32,
+    ) -> Result<MutationResult, Error> {
+        let mut result = self.multi_mutate(state, input, stage_idx, Some(1))?;
+        if let Some(first_input) = result.pop() {
+            *input = first_input;
```

Comment:
nono we return.a vector here
we want the entire `result` vector

Reply:
if result.len() > 0 => mutated else skipped

Reply:
Yes and a single mutate returns the first element of the vector

Reply:
But I can remove the single mutate if you don't think that makes sense

Reply:
Removed